examples/gallery/index.html

```diff
@@ -36,11 +43,38 @@ <h1>Slint Gallery</h1>
     </a>
   </p>
   <script type="module">
-    import init from './pkg/gallery.js';
-    init().finally(() => {
-      document.getElementById("spinner").remove();
-    });
+    var fluent = `./pkg/fluent/gallery.js`;
+    var material = "./pkg/material/gallery.js";
+
+    function initGallery(gallery) {
+      document.getElementById("spinner").hidden = false;
+      document.getElementById("canvas").hidden = true;
```

Comment:
Hmm, doesn't this hide the spinner right away on startup, when we probably want it?

Reply:
What do you mean?

Reply:
Ahh, never mind, I misread `hidden = false` and `hidden = true` :)

Reply:
It does not hide it on startup. It set  `hidden` to false. The spinner is only hidden after the gallery is loaded.